tasks/sentry.sh

```diff
@@ -1,6 +1,6 @@
 # install sentry cli
 echo "Installing and configuring sentry cli"
-curl -L -o sentry-cli https://github.com/getsentry/sentry-cli/releases/download/1.40.0/sentry-cli-Linux-x86_64
+curl -L -o sentry-cli https://github.com/getsentry/sentry-cli/releases/download/1.40.0/sentry-cli-Linux-`arch`
```

Comment:
Why does it need to be a string like that for `arch` and not for `x86_64`?

Reply:
It's not just a string, it's a command. Placing it in ` means that we need to put here its result

Reply:
oh I see. And what does the command `arch` then do? Fetch the `arch` version?

Reply:
Yap, it prints computer architecture.

Reply:
Nice!